src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1744,17 +1749,22 @@ def _aks_browse(
         logger.warning('Proxy running on %s', proxy_url)
 
     timeout = None
-    if os.getenv("PYTEST_CURRENT_TEST", None):
+    if "test_aks_browse_legacy" in os.getenv("PYTEST_CURRENT_TEST", "").lower():
```

Comment:
is it better to setup a test hook config file instead of env variable?

Reply:
changed in commit [42c9405](https://github.com/Azure/azure-cli/pull/19115/commits/42c9405de2e92af7de4d2f3191ff37dc2c24ab4f).